content/shmem_init_thread.tex

```diff
@@ -0,0 +1,53 @@
+\apisummary{
+Initializes the OpenSHMEM library, similar to \FUNC{shmem\_init}, and performs any
+initialization required for supporting the four thread levels.}
```

Comment:
Remove explicit enumeration "four" in case future thread levels are added. Check for other instances.